### ISSUE TITLE
Alert created from REST API: Ability to review alert config from UI in read-only mode.

### ISSUE THREAD

**Issue by harish422** [grafana/grafana#65647]
Hi Grafana Team,

I noticed for any alerts created from REST APIs "Alertingprovisioning" module, they can't be edited or alert config can't be viewed from UI. I understand why write access should be cut down in UI.
But as an enhancement, is it possible for atleast to make the alert config(queries etc) to be visible from UI , but just read-only ?
right now we(and other) are finding it a bit hard to follow the alert config from UI for alerts created from REST API, maintaining the alert, config can only be done from API.
Not fair for "Anyone/everyone" to call(GET) the api's to quickly review the alert config. might not be their cup of tea.
This all what we can see right now. But reviewing the alert config is important to us.

![Screen Shot 2023-03-30 at 11 48 54 AM](https://user-images.githubusercontent.com/23487589/228936550-9294d895-b313-4b5d-82a1-8b1a2ebc1f86.png)

**Comment by tonypowa** [comment]
thanks for creating this issue @harish422 

you can edit, and review, API provisioned alert rules in the UI as long as you have included the header `x-disable-provenance` when creating the rules via your (POST) request

More details here:
https://grafana.com/docs/grafana/latest/developers/http_api/alerting_provisioning/#span-idroute-post-alert-rulespan-create-a-new-alert-rule-_routepostalertrule_

**Comment by grobinson-grafana** [comment]
Seems reasonable to me! @soniaAguilarPeiron would you like to work on this together in the coming weeks?

**Comment by armandgrillet** [comment]
On Grafana 10.0, clicking on Actions => "View" provide the ability to see the entire alert configuration, including its queries and expressions. Please reopen if this is not what you need.
